parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetReader.java

```diff
@@ -115,9 +115,9 @@ private ParquetReader(Configuration conf,
         readSupport);
   }
 
-  private ParquetReader(List<InputFile> files,
-                        ParquetReadOptions options,
-                        ReadSupport<T> readSupport) throws IOException {
+  public ParquetReader(List<InputFile> files,
```

Comment:
Since it is going be public, please add comments like above.

Reply:
Done